core/iwasm/libraries/lib-wasi-threads/test/build.sh

```diff
@@ -9,6 +9,36 @@ set -eo pipefail
 CC=${CC:=/opt/wasi-sdk/bin/clang}
 WAMR_DIR=../../../../..
 
+rm -f *.aot
+rm -f *.wasm
+
+# Set default values
+enable_stress_tests=false
+
+# Function to display usage information
+function display_usage() {
+    echo "Usage: $0 [--enable_stress_tests]"
+    echo "Options:"
+    echo "  --enable_stress_tests   Enable stress tests"
+}
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --enable_stress_tests)
```

Comment:
Why do we need it? I see you already exclude tests by defining them in `skip.json` file so it's probably fine to compile them unconditionally?

Reply:
I think we can control tests that are/aren't included using wasi tests' filters